rclcpp/src/rclcpp/executor.cpp

```diff
@@ -218,9 +222,11 @@ Executor::spin_once(std::chrono::nanoseconds timeout)
   if (spinning.exchange(true)) {
     throw std::runtime_error("spin_once() called while already spinning");
   }
-  RCLCPP_SCOPE_EXIT(this->spinning.store(false); );
-  auto any_exec = get_next_executable(timeout);
-  if (any_exec) {
+  RCLCPP_SCOPE_EXIT({
+    this->spinning.store(false);
+  });
```

Comment:
Just curious, is this a bug fix or style fix? It looks like the macro already adds `{}` around its arguments.

Reply:
Style only.

Reply:
For consistency, there's another use of the scope exit macro on line 208 that could have `{}` added to it.

Reply:
I'll remove this diff and maybe make a separate pr for this change then, since its growing in scope :)